webpack/JobInvocationDetail/TemplateInvocationComponents/TemplateActionButtons.js

```diff
@@ -47,7 +47,8 @@ const actions = ({
         APIActions.post({
           url: `/foreman_tasks/tasks/${taskID}/cancel`,
           key: 'CANCEL_TASK',
-          errorToast: ({ response }) => response.data.message,
+          errorToast: ({ response }) =>
+            response.data.message || __('Could not cancel the task'),
```

Comment:
Should this use safe navigation as well?

Reply:
by safe navigation, do you mean ` response?.data?.message`?

Reply:
Yes